xml/release-notes.xml

```diff
@@ -95,6 +95,44 @@
    Make sure to also review <xref linkend="sec.driver"/>.
   </para> -->
 
+  <sect2 xml:id="installation-new-update-repos">
+   <title>&opensuseleap; has newly three update repos</title>
+   <!-- boo#1186593 -->
+   <para>
+    Since &opensuseleap; 15.3 the maintenance setup consists of three main update repositories.
+    These are repo-update, repo-backports-update, and repo-sle-update. The latter two are brand new and
+    are a result of reworking distribution to re-use binaries from SUSE Linux Enterprise.
+
+    These repositories, are available and checked during the Online installation of &opensuseleap;.
+    We would like to kindly ask users not to uncheck them.
+    New update repo definitions for &opensuseleap; 15.3 will be additionally supplied via a 0day maintenance update of the <package>openSUSE-release</package>.
+
+    The update will be delivered via the traditional maintenance channel (repo-update).
+    It will carry a special update flag, to signalize that it touches the software management area which is then specially handled by zypp.
+    Users are advised to double-check with additional <command>zypper up</command> whether all updates were processed.
+   </para>
+   <link xlink:href="ihttps://bugzilla.opensuse.org/show_bug.cgi?id=1186593"/>.
+   <para>
+    The repo-sle-update repository is for openSUSE Leap (OSS) updates, this is the smallest one,
+    which contains system configuration packages including Release package, branding, and potential forks of SLE packages.
+    This repository has also a debug-info variant.
+   </para>
+   <para>
+    The repo-backports-update is an update repository for openSUSE Backports that contains updates for the majority of Leap packages.
+    This repository has also a debug-info variant.
```

Comment:
Minor nitpicking: I'd write this as `This repository also has` (instead of `has also`). If in doubt, ask a native english speaker which is better / more correct.

Reply:
Fixed in next commit